InvestmentCompany.java

```diff
@@ -0,0 +1,168 @@
+package com.company;
+
+public class InvestmentCompany {
+    private String address;
+    private int investments;
+    private String capital;
+
+    private String country;
+    private String name;
+    private int employees;
+    private String founder;
+    private String reliability;
+
+    private static int yearOfFoundation = 1960;
```

Comment:
Still, this static field makes no sense because multiple investment companies can have different years of foundation, static fields are bound to the class but not to the object, so try to change it to something that refers to all Investment companies as a whole (for example average employee count of all investment companies)